src/core/reducers/addons.js

```diff
@@ -24,16 +25,16 @@ type ExternalAddonMap = {
 };
 
 export type LoadAddonsAction = {|
-  payload: {| addons: ExternalAddonMap |},
+  payload: {| addons: ExternalAddonMap | ExternalDiscoAddonMap |},
   type: typeof LOAD_ADDONS,
 |};
 
 // TODO: We should remove this method and move all calls to `loadAddonResults`.
 // This function relies on normalizr messing with our response data.
 // See: https://github.com/mozilla/addons-frontend/issues/2917
-export function loadAddons(entities: {|
-  addons?: ExternalAddonMap,
-|}): LoadAddonsAction {
+export function loadAddons(entities: {
+  addons: ExternalAddonMap,
```

Comment:
should this be ` ExternalAddonMap | ExternalDiscoAddonMap` too?

Reply:
ah good catch, added, thanks!